### PR TITLE
Es6 refactor sync async file system decorator

### PR DESCRIPTION
**Pull request by rachel-ftw**
Line 6 of this is confusing...
Linter says `fs` isn't used, but it visably is... but when commented out tests still pass. I left it commented out.
Not sure what to do with this.

### CODE REVIEW

**Comment by rachel-ftw** [comment]
deleted the line library that referenced the file system... 
@TheLarkInn would this work for me bc of my version of node? ...? Not wanting to fuck it up, but not knowing the details on this.

**Comment by TheLarkInn** [comment]
interesting on the diff it doesn't even show that line being there, maybe was deleted from when originally was merged

File: lib/SyncAsyncFileSystemDecorator.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
+"use strict";
```
Comment (sokra) [inline lib/SyncAsyncFileSystemDecorator.js]: use strict after comment

**Comment by ooflorent** [comment]
Thank you for your PR but it appears that the file was previously converted to ES6. I'm closing the PR for this reason.

**Pull request closed without merging**